packages/contracts/test/voting/addresslist-voting.ts

```diff
@@ -736,25 +726,29 @@ describe('AddresslistVoting', function () {
         }
 
         // calling execute again should fail
-        await expect(voting.execute(id)).to.be.revertedWith(
-          customError('ProposalExecutionForbidden', id)
-        );
+        await expect(voting.execute(id))
+          .to.be.revertedWithCustomError(voting, 'ProposalExecutionForbidden')
+          .withArgs(id);
       });
 
       it('reverts if vote is not decided yet', async () => {
         await advanceIntoVoteTime(startDate, endDate);
 
-        await expect(voting.execute(id)).to.be.revertedWith(
-          customError('ProposalExecutionForbidden', id)
-        );
+        await expect(voting.execute(id))
+          .to.be.revertedWithCustomError(voting, 'ProposalExecutionForbidden')
+          .withArgs(id);
       });
     });
 
     context('Vote Replacement Mode', async () => {
       beforeEach(async () => {
         votingSettings.votingMode = VotingMode.VoteReplacement;
 
-        await voting.initialize(dao.address, votingSettings, addresslist(10));
+        await voting.initialize(
+          dao.address,
+          votingSettings,
+          [0, 1, 2, 3, 4, 5, 6, 7, 8, 9].map(i => signers[i].address)
```

Comment:
using `slice(...)` with `map(...)` would probably be cleaner: `signers.slice(0, 9).map(s => s.address)`